tutorials/developer-tutorials/createNewIdentity.md

```diff
@@ -0,0 +1,23 @@
+## **Create New Identity Using Local Fabric CA**
+`2-3 mins`
+
+> **Important:** You should have a **smart contract** instantiated on your Local Fabric to follow this tutorial. Follow part 1 of this series first for instructions.
+
+
+## Learning Objectives
+
+* Create a new identity (no attributes) using Local Fabric CA
+
+---
+<details>
+<summary><b>1. Create a new identity</b></summary>
+
+In the very first tutorial (Tutorial 1: Local smart contract development), we used a pre-set identity (`admin`) to connect to the Local Fabric gateway and submit/evaluate transactions. In this tutorial, we will learn how to create _our own_ identity.
+
+Having multiple identities enrolled with the Local Fabric CA (Certificate Authority) allows you to represent different participants in a blockchain network. For example two identities `Jack` (the customer) and `Tesla` (the manufacturer) may be permitted to submit different transactions in a smart contract.
+
+1. In the `FABRIC ENVIRONMENTS` panel, look for `ca.org1.example.com` (it's under Nodes), right click it and choose `Create Identity (register and enroll)`. 
```

Comment:
you need to tell them to start the local fabric and connect to it first

Reply:
Fixed with by editing step one to:
Under the `FABRIC ENVIRONMENTS` panel, click on `Local Fabric  ○ (click to start)` to start the local fabric. Once it is started, look for `ca.org1.example.com` (it's under Nodes), right click it and choose `Create Identity (register and enroll)`.